src/main/java/uk/gov/hmcts/darts/common/repository/CaseRepository.java

```diff
@@ -24,4 +24,10 @@ and upper(case.courthouse.courthouseName) = upper(:courthouseName)
         """)
     List<String> findOpenCaseNumbers(String courthouseName, List<String> caseNumbers);
 
+    @Query("""
```

Comment:
Would an `EXISTS` query be possible and more appropriate for performance reasons? As the system ages, the number of case associated with a courthouse will become significant, and `COUNT` may involve a full table scan, wheres `EXISTS` will stop after finding the first match.